### PR TITLE
Explain the use of application/vc|vp in envelopes.

### PR DESCRIPTION
**Pull request by BigBlueHat**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1546.html" title="Last updated on Aug 16, 2024, 4:01 PM UTC (b0feaaf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1546/4c6005d...b0feaaf.html" title="Last updated on Aug 16, 2024, 4:01 PM UTC (b0feaaf)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7216,8 +7216,8 @@ <h2>application/vc</h2>
         </p>
 
         <p>
-This media type can be used for credentials secured using an
-[=enveloping proof=].
+This media type can be used within [=enveloping proof=] when denoting the
```
Comment (dlongley) [inline index.html:7219]:
```suggestion
This media type can be used within [=enveloping proofs=] when denoting the
```

File: index.html [hunk 2]
```diff
@@ -7276,8 +7276,8 @@ <h2>application/vp</h2>
         </p>
 
         <p>
-This media type can be used for presentations secured using an
-[=enveloping proof=].
+This media type can be used within [=enveloping proof=] when denoting the
```
Comment (dlongley) [inline index.html:7279]:
```suggestion
This media type can be used within [=enveloping proofs=] when denoting the
```

**Review by dlongley — APPROVED** [review]
Minor grammar fix.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:7219]: Given `contents of the [singular] envelope` that follows, I think keeping the `proof` singular makes sense. Adding `an` fixes the grammar.
```suggestion
This media type can be used within an [=enveloping proof=] when denoting the
```

File: index.html (hunk 2 above)
Comment (TallTed) [reply index.html:7279]:
```suggestion
This media type can be used within an [=enveloping proof=] when denoting the
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by iherman — APPROVED** [review]
Approving once the comments of @TallTed are incorporated.

**Review by selfissued — CHANGES_REQUESTED** [review]
The new wording is longer and harder to read.

**Comment by TallTed** [comment]
> [@selfissued] The new wording is longer and harder to read.

The prior wording was unclear. Concrete suggestions of further revision would enable their consideration.

File: index.html [hunk 3]
```diff
@@ -7216,8 +7216,8 @@ <h2>application/vc</h2>
         </p>
 
         <p>
-This media type can be used for credentials secured using an
-[=enveloping proof=].
+This media type can be used within an [=enveloping proof=] when denoting the
+contents of the envelope.
```
Comment (dlongley) [inline index.html:7220]: Shortening the text and making it easier to read (to address Mike's concern):

```suggestion
This media type can be used in an [=enveloping proof=] to denote the enveloped payload.
```

File: index.html [hunk 4]
```diff
@@ -7276,8 +7276,8 @@ <h2>application/vp</h2>
         </p>
 
         <p>
-This media type can be used for presentations secured using an
-[=enveloping proof=].
+This media type can be used within an [=enveloping proof=] when denoting the
+contents of the envelope.
```
Comment (dlongley) [inline index.html:7280]: Shortening the text and making it easier to read (to address Mike's concern):

```suggestion
This media type can be used in an [=enveloping proof=] to denote the enveloped payload.
```

**Comment by dlongley** [comment]
@selfissued, we tried to address your concern and requested a re-review from you, thanks!

**Review by msporny — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Pull request merged**